nautobot/extras/jobs.py

```diff
@@ -539,7 +539,7 @@ def serialize_data(data):
             elif isinstance(value, Model):
                 return_data[field_name] = value.pk
             # FileVar (Save each FileVar as a FileProxy)
-            elif isinstance(value, InMemoryUploadedFile):
+            elif isinstance(value, UploadedFile):
```

Comment:
The bug occurred because we only checked for `InMemoryUploadedFile` types, missing `TemporaryUploadedFile` types for large CSV files.